src/Chapter1.hs

```diff
@@ -541,8 +541,13 @@ value after "=" where the condition is true.
 
 Casual reminder about adding top-level type signatures for all functions :)
 -}
-
-mid x y z = error "mid: not implemented!"
+mid :: Int -> Int -> Int -> Int
+mid x y z 
+  | (x > y && x < z) || (x < y && x > z) = x
+  | (y > x && y < z) || (y < x && y > z) = y
+  | x == y || x == z                     = x
+  | y == z                               = y
```

Comment:
Great guards! 💯 

I think these two cases could be covered if you would use `>=` and `<=` in the comparisons above 🙂